src/main/java/Router.java

```diff
@@ -18,7 +23,7 @@ private Handler getHandler(String uri) {
       return this.routes.get(uri);
     } 
 
-    return new NotFoundHandler();
+    return defaultHandler;
```

Comment:
This means that a default handler should be able to respond with not found, right?

Reply:
Correct.